akka-docs-dev/rst/java/stream-quickstart.rst

```diff
@@ -136,26 +156,30 @@ While this question is not as obvious to give an answer to in case of an infinit
 this question in a streaming setting would to create a stream of counts described as "*up until now*, we've processed N tweets"),
 but in general it is possible to deal with finite streams and come up with a nice result such as a total count of elements.
 
-First, let's write such an element counter using :class:`FoldSink` and then we'll see how it is possible to obtain materialized
-values from a :class:`MaterializedMap` which is returned by materializing an Akka stream. We'll split execution into multiple
-lines for the sake of explaining the concepts of ``Materializable`` elements and ``MaterializedType``
+First, let's write such an element counter using `Flow.of(Class)`` and ``Sink.fold`` to see how the types look like:
 
 .. includecode:: ../../../akka-samples/akka-docs-java-lambda/src/test/java/docs/stream/TwitterStreamQuickstartDocTest.java#tweets-fold-count
 
-First, we prepare the :class:`FoldSink` which will be used to sum all ``Integer`` elements of the stream.
-Next we connect the ``tweets`` stream though a ``map`` step which converts each tweet into the number ``1``,
-finally we connect the flow ``to`` the previously prepared Sink. Notice that this step does *not* yet materialize the
-processing pipeline, it merely prepares the description of the Flow, which is now connected to a Sink, and therefore can
-be ``run()``, as indicated by its type: :class:`RunnableGraph`. Next we call ``run()`` which uses the implicit :class:`ActorMaterializer`
-to materialize and run the flow. The value returned by calling ``run()`` on a ``RunnableGraph`` or ``FlowGraph`` is ``MaterializedMap``,
-which can be used to retrieve materialized values from the running stream.
+First we prepare a reusable ``Flow`` that will change each incoming tweet into an integer of value ``1``.
+We'll use this in order to combine those ones with a ``Sink.fold`` will sum all ``Int`` elements of the stream

```

Comment:
We combine all values of transformed stream using a ``Sink.fold`` that sums all ...